.env.example

```diff
@@ -49,9 +51,9 @@ MINIO_ADMIN_PORT=9001
 
 # PostgreSQL
 PG_PORT=7001
-PG_DATABASE=👻
+PG_DATABASE=planxdb
+PG_USERNAME=dbuser
```

Comment:
this is already exposed in infrastructure/data too

Reply:
_nit_: I'm maybe a little bit paranoid but I'd lean towards an abundance of caution removing them from  infrastructure/data and not exposing them here. Although as I say it's more of a feeling than anything else 😅

Reply:
This is a fair comment! A bit more context: this database name is only used in Docker envs (which will never have "real" session data), our staging & production databases on AWS RDS have different names. The username is shared across envs but has already been exposed for awhile and isn't sensitive on its own and was annoyingly required to successfully spin up the DB. 

Security-wise, I think working towards more restricted RDS access altogether will give us a much better peace of mind!

Reply:
That makes a lot of sense, thanks for the extra context 🙌